### PR TITLE
Ansible developer tools personas

### PR DESCRIPTION
**Pull request by oraNod**
Adds persona definitions for Ansible developer tool documentation. The purpose of these personas is specifically intended to help identify the target audience for technical content, such as the docs.

### CODE REVIEW

File: docs/journeys/personas.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- **Knowledge:** Strong knowledge of fundamentals of computing. Uses an IDE as part of the daily workflow.
+
+# IT automation engineer
+
+I create automation content, composed of multiple tasks or playbooks, that runs against single platforms.
+I want to reuse automation content where possible and share automation content with others.
+I work independently or with a small team and have an innovative mindset with strong problem-solving ability.
+
+- **Needs:** Task-oriented content focused on building and testing automation content. Conceptual details related to organizing and structuring projects. Extensive reference content that explains "if you do x, then y is the expected behaviour".
+- **Attitude:** Wants to understand how things work and what are the available options.
```
Comment (ariordan-redhat) [inline docs/journeys/personas.md:25]:
```suggestion
- **Attitude:** Wants to understand how things work and what the available options are.
```

File: docs/journeys/personas.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- **Knowledge:** Strong knowledge of CI/CD platforms and tools. Expert with scripting languages.
+
+# Process automation engineer
+
+I create end-to-end automation solutions across multiple services or applications.
+I also manage things like inventories and credentials and work with cross-functional teams to oversee testing, delivery, and validation of solutions.
+In my role I use CI/CD pipelines to provision cloud resources and integrate solutions on a continual basis. Creates separate stage and production environments.
```
Comment (ariordan-redhat) [inline docs/journeys/personas.md:32]:
```suggestion
In my role I use CI/CD pipelines to provision cloud resources and integrate solutions on a continual basis. I create separate stage and production environments.
```

**Comment by trishnaguha** [comment]
@oraNod This needs to reviewed by @tima before this is published. Whether the upstream Docs should have it or not.

**Comment by tima** [comment]
@oraNod How do these get resolved with the internal personas written up for dev tools in ANSTRAT-119?

**Comment by oraNod** [comment]
> @oraNod How do these get resolved with the internal personas written up for dev tools in ANSTRAT-119?

@tima I wasn't aware of ANSTRAT-119. Looking through I think the personas we've defined here generally map to the ones in that issue.

Novice = Learner persona
IT automation engineer + Process automation engineer = Enterprise developer
Automation developer = ISV content creator
Operations engineer = CI/CD (more or less)

The point for the personas here is to help out with the documentation moreso than anything else. If these personas conflict with the ones in ANSTRAT-119 then maybe we should find another repo to store them in? Alternatively I guess we could rewrite the personas to match the ones in that issue but they sound much more strictly "enterprise".

**Comment by oraNod** [comment]
Closing this PR after some discussion with @trishnaguha and @cidrblock Thanks everyone.

**Pull request closed without merging**